content/id/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Disrupsi
+content_template: templates/concept
+weight: 60
+---
+
+{{% capture overview %}}
+Petunjuk ini diperuntukkan bagi pemilik aplikasi yang ingin membuat aplikasi yang memiliki ketersediaan yang tinggi, sehingga dibutuhkan untuk mengerti jenis-jenis Gangguan yang dapat terjadi pada Pods.
+
+Petunjuk ini juga diperuntukkan bagi administrator kluster yang ingin melakukan berbagai tindakan otomasi pada kluster, seperti pembaruan dan *autoscaling* kluster.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Disrupsi yang Disengaja dan Tidak Disengaja
+
+Pods tidak akan terhapus hingga sesuatu (orang maupun *controller*) menghancurkan mereka atau ada kesalahan perangkat keras maupun perangkat lunak yang tidak dapat dihindari.
+
+Kita menyebut kasus-kasus yang tidak dapat dihindari sebagai *disrupsi yang tidak disengaja* terhadap aplikasi. Beberapa contohnya adalah sebagai berikut:
+
+- Kesalahahan perangkat keras pada mesin yang menjalankan Node
+- Administrator kluster menghapus *virtual machine* secara tidak sengaja
+- Kesalahan pada penyedia layanan cloud yang mengakibatkan terhapusnya *virtual machine*
+- Sebuah *kernel panic*
+- Node menghilang dari kluster karena partisi jaringan kluster
+- Diusirnya Pod karena Node [kehabisan sumber daya](/docs/tasks/administer-cluster/out-of-resource)
+
+Dengan pengecualian pada kondisi kehabisan sumber daya, kondisi-kondisi tersebut pada umumnya diketahui oleh kebanyakan pengguna; Kondisi-kondisi tersebut tidak hanya terjadi pada Kubernetes
+
+Kita menyebut kasus-kasus lainnya sebagai *disrupsi yang disengaja*. Hal ini termasuk tindakan yang dilakukan oleh pemilik aplikasi atau yang dilakukan oleh administrator kluster. Pemilik aplikasi umumnya melakukan hal-hal berikut:
+
+- Menghapus Deployment atau pengontrol yang mengatur Pod
+- Memperbarui templat Pod yang menyebabkan pengulangan kembali/*restart*
+- Menghapus Pod secara langsung
+
+Administrator kluster umumnya melakukan hal-hal berikut:
+
+- [Menguras Node](/docs/tasks/administer-cluster/safely-drain-node/) untuk perbaikan atau pembaruan.
+- Menguras sebuah node dari kluster untuk memperkecil ukuran kluster (untuk lebih lanjutnya, pelajari [*Autoscaling kluster*](/docs/tasks/administer-cluster/cluster-management/#cluster-autoscaler)).
+- Menghapus sebuah Pod dari node untuk memuat Pod lain ke node tersebut.
+
+Tindakan-tindakan tersebut dapat dilakukan secara langsung oleh administrator kluster, atau oleh alat otomasi yang dijalankan oleh administrator kluster, atau oleh penyedia layanan cloud.
+
+Tanyakan administrator kluster atau penyedia layanan cloud anda, atau lihatlah dokumentasi penyedia layanan cloud anda untuk mengetahui bila ada sumber-sumber yang berpotensi mengakibatkan disrupsi yang disengaja yang ada pada kluster anda. Jika tidak ada, anda bisa melewatkan pembuatan *PodDisruptionBudget*
```

Comment:
prefer "kamu" over "anda", to align with Kubernetes community philosophy of being friendly

apply this to rest of the page